### PR TITLE
feat: add new cache invalidation content

### PR DESCRIPTION
**Pull request by ankur-arch**
closes #DA-1642

### CODE REVIEW

File: content/300-accelerate/600-faq.mdx [hunk 1]
```diff
@@ -182,3 +182,17 @@ As the cache needs to be cleared globally, it is difficult to provide a specific
 **Invalidate**: The cache entry is deleted, and new data will be fetched on the next request, causing a cache miss. This removes stale data but may lead to slower responses until the cache is repopulated.
 
 **Revalidate**: The cache entry is updated proactively, ensuring the next request uses fresh data from the cache. This keeps the cache valid and maintains faster response times by avoiding cache misses.
+
+### What is on-demand cache invalidation?
+
+[On-demand cache invalidation](/accelerate/caching#on-demand-cache-invalidation) lets applications instantly update specific cached data when it changes, instead of waiting for regular cache refresh cycles. This keeps information accurate and up-to-date for users.
+
+#### When should I use the cache invalidate API?
```
Comment (nikolasburk) [inline content/300-accelerate/600-faq.mdx]: Is it intentional that these are H3 and H4 headings? Shouldn't all Qs on this page be H2 headings?

MDX source: 

image

Preview: 

image

**Review by nikolasburk — COMMENTED** [review]
This is great Ankur, I really like the way how you weaved in the cache invalidation info. The examples and code snippets you chose are also very clear, nice work 👌 

Just one Q about the H2/H3/H4 headings on the FAQ page. Once that's cleared, this should be good to get merged 🙌

File: content/300-accelerate/600-faq.mdx (hunk 1 above)
Comment (ankur-arch) [reply content/300-accelerate/600-faq.mdx]: Thanks for the quick review @nikolasburk. This was an oversight on my part. I resolved it.

File: content/300-accelerate/600-faq.mdx [hunk 2]
```diff
@@ -182,3 +182,17 @@ As the cache needs to be cleared globally, it is difficult to provide a specific
 **Invalidate**: The cache entry is deleted, and new data will be fetched on the next request, causing a cache miss. This removes stale data but may lead to slower responses until the cache is repopulated.
 
 **Revalidate**: The cache entry is updated proactively, ensuring the next request uses fresh data from the cache. This keeps the cache valid and maintains faster response times by avoiding cache misses.
+
+## What is on-demand cache invalidation?
+
+[On-demand cache invalidation](/accelerate/caching#on-demand-cache-invalidation) lets applications instantly update specific cached data when it changes, instead of waiting for regular cache refresh cycles. This keeps information accurate and up-to-date for users.
+
+### When should I use the cache invalidate API?
```
Comment (ankur-arch) [inline content/300-accelerate/600-faq.mdx:190]:
```suggestion
## When should I use the cache invalidate API?
```

**Review by mhessdev — APPROVED** [review]

**Pull request merged**